inception/inception-ui-project/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/project/layers/ProjectLayersPanel.properties

```diff
@@ -15,10 +15,10 @@
 # limitations under the License.
 
 help.label=Help
-layers.help.link=doc/user-guide.html#sect_projects_layers
-properties.help.link=doc/user-guide.html#sect_projects_layers_properties
-behaviours.help.link=doc/user-guide.html#sect_projects_layers_behaviours
-features.help.link=doc/user-guide.html#sect_projects_layers_features
+layers.help.link=../../../doc/user-guide.html#sect_projects_layers
```

Comment:
This works when accessing the layer settings through the project dashboard with URLs of this form

```
http://localhost:8080/inception-app-webapp/p/394/settings/layers
```

But it doesn't work when accessing the settings via the Administrator -> Projects path which has these URLs:

```
http://localhost:8080/inception-app-webapp/admin/projects
```